src/main/java/com/diffmin/patch/PatchApplication.java

```diff
@@ -80,4 +82,13 @@ private static void performInsertion(
                 break;
         }
     }
+
+    private static void performMovement(
+            Pair<CtElement, ImmutableTriple<Integer, CtElement, CtElement>> movePatch) {
+        CtElement toBeDeleted = movePatch.getFirst();
+        ImmutableTriple<Integer, CtElement, CtElement> toBeInserted = movePatch.getSecond();
+
+        performDeletion(toBeDeleted);
+        performInsertion(toBeInserted);
```

Comment:
Should we care to enclose this in a `try-catch` block? Suppose if `performDeletion` fails, we would not want to execute `performInsertion` just for the sake of atomicity.